Misc/NEWS.d/next/IDLE/2019-11-23-21-50-57.bpo-38862.KQ9A0m.rst

```diff
@@ -0,0 +1,4 @@
+Format menu Strip Trailing Whitespace adjusts ending newlines of non shell
+windows so that the file is either blank or ends with one newline after
```

Comment:
Perhaps "or ends with a single newline" is sufficient, given that "Strip Trailing Whitespace" by definition removes trailing spaces and tabs of each line? In other words, I'm not sure this level of technical detail is appropriate here.

Reply:
After changing the issue spec, this simplifies to 'remove extra newlines'.